src/iris/api.py

```diff
@@ -3920,91 +3922,97 @@ def on_get(self, req, resp):
+    user_id = user_data.pop('id')
+
+    # get any mode based template override settings
+    mode_template_override_query = '''SELECT `mode`.`name` AS `mode`
+                        FROM `mode_template_override`
+                            JOIN `mode` ON `mode`.`id` = `mode_template_override`.`mode_id`
+                        WHERE `mode_template_override`.`target_id` = %s'''
+    cursor.execute(mode_template_override_query, user_id)
+    user_data['template_overrides'] = [row['mode'] for row in cursor]
+
+    # get any category override settings
+    category_override_query = '''SELECT `application`.`name` AS `application_name`, `mode`.`name` AS `mode`, `notification_category`.`name` AS `category`
+                                    FROM `category_override`
+                                    JOIN `mode` ON `mode`.`id` = `category_override`.`mode_id`
+                                    JOIN `notification_category` ON `notification_category`.`id` = `category_override`.`category_id`
+                                    JOIN `application` ON `application`.`id` = `notification_category`.`application_id`
+                                    WHERE `category_override`.`user_id` = %s'''
+    cursor.execute(category_override_query, user_id)
+    user_data['category_overrides'] = {}
+    for row in cursor:
+        user_data['category_overrides'][row['application_name']] = row
+
+    # Get user contact modes
+    modes_query = '''SELECT `priority`.`name` AS priority, `mode`.`name` AS `mode`
+                        FROM `target` JOIN `target_mode` ON `target`.`id` = `target_mode`.`target_id`
+                            JOIN `priority` ON `priority`.`id` = `target_mode`.`priority_id`
+                            JOIN `mode` ON `mode`.`id` = `target_mode`.`mode_id`
+                        WHERE `target`.`id` = %s'''
+    cursor.execute(modes_query, user_id)
+    user_data['modes'] = {}
+    for row in cursor:
+        user_data['modes'][row['priority']] = row['mode']
+
+    # get device settings for user
+    mode_template_override_query = '''SELECT `device`.`registration_id`, `device`.`user_id`, `device`.`platform`
+                        FROM `device`
+                        WHERE `device`.`user_id` = %s'''
+    cursor.execute(mode_template_override_query, user_id)
+    user_data['device'] = [row for row in cursor]
+
+    # Get user contact modes per app
+    user_data['per_app_modes'] = defaultdict(dict)
+    cursor.execute(get_all_users_app_modes_query, user_id)
+    for row in cursor:
+        user_data['per_app_modes'][row['application']][row['priority']] = row['mode']
+
+    # Get user teams
+    teams_query = '''SELECT `target`.`name` AS `team`
+                    FROM `user_team` JOIN `target` ON `user_team`.`team_id` = `target`.`id`
+                    WHERE `user_team`.`user_id` = %s'''
+    cursor.execute(teams_query, user_id)
+    user_data['teams'] = []
+    for row in cursor:
+        user_data['teams'].append(row['team'])
```

Comment:
Like on 3949, can this be simplified to the following?

    user_data['teams'] = [row['team'] for row in cursor]